src/structures/song_selector.ts

```diff
@@ -225,19 +224,23 @@ export default class SongSelector {
         this.uniqueSongsPlayed.clear();
     }
 
-    getSongs(): { songs: Set<QueriedSong>; countBeforeLimit: number } {
-        if (!this.filteredSongs) {
+    getSongs(): SelectedSongs {
+        if (!this.selectedSongs) {
             return {
                 songs: new Set(),
                 countBeforeLimit: 0,
```

Comment:
This should also have ineligibleDueToCommonAlias

Reply:
its nullable